### PR TITLE
AC Automated Test - LRAC-8568 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LRAC-8568
https://issues.liferay.com/browse/LRAC-9332 (refactor ticket)
https://issues.liferay.com/browse/LRAC-9263 (bug ticket)

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 1]
```diff
@@ -271,12 +271,76 @@ definition {
...
+			ACSegments.selectDocument(
+				documentTitle = "userfn userln",
+				indexField = "1");
+		}
+
+		task ("View and asset user") {
+			ACSegments.previewSegmentMembership();
+
+			ACAssets.viewAssets(assetName = "userfn userln");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:343]: Hi @andreefarias, I think you can save the segment and then open the segment and at this moment you can check the user

Comment (andreefarias) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:343]: Ok, that's right!

**Comment by andreefarias** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @andreefarias , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**